tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -973,6 +980,22 @@ void SPIRVEmitter::doFunctionDecl(const FunctionDecl *decl) {
   continueStack = std::stack<uint32_t>();
 
   std::string funcName = decl->getName();
+  std::string nsPrefix = "";
+
+  // Add namespace name as prefix of function name (if any).
+  const DeclContext *dc = decl->getEnclosingNamespaceContext();
+  while (dc && !dc->isTranslationUnit()) {
```

Comment:
It seems we can use FunctionDecl library functions to get the qualified name?
E.g., https://clang.llvm.org/doxygen/classclang_1_1FunctionDecl.html#af40656355e6d13ca9d7de07340b1b46c

But no need to fix in this PR. Can handle it in the next PR.